### ISSUE TITLE
tracking: Skyra.Image service

### ISSUE THREAD

**Issue by kyranet** [skyra-project/skyra#1537]
| ⚠️ Once SAEVII is finished, this service should be merged into the core to minimise bandwidth in the future, this issue is to seek an alternative to node-canvas to ensure full functionality on release. |
| --- |

.NET does not have bindings for Cairo nor Pango (the libraries used for node-canvas), furthermore, they are very troublesome and perform poorly compared to other alternatives in .NET.

This service will first be outside the bot, using gRPC, but once the bot is written in .NET, a last step will be taken to remove gRPC and integrate the service into the bot to minimise overhead in the future. As such, the API should be designed for reusability: it should be easy to replace the gRPC layer with direct calls, and no changes in the functionality of the library should be required to accomplish this.

### Progress

- [ ] Creating Image framework.
- [ ] Creating internal API with gRPC to send commands and receiving buffers back.
